src/libraries/System.Private.CoreLib/src/System/Number.BigInteger.cs

```diff
@@ -812,10 +795,6 @@ public static void Multiply(ref BigInteger lhs, ref BigInteger rhs, ref BigInteg
                 {
                     result._length = (maxResultLength - 1);
```

Comment:
this could probably be simplified to `result._length -= 1` now.

Reply:
Done; used -- for code style conformance.